### DISCUSSION TITLE
Party chat along with action?

### DISCUSSION THREAD

**Discussion by Andorahfal** [Technyze/XIVHotbar2#3] [Q&A] [answered]
Is there a way to also say something in Party chat along with an action? For example to say that I am provoking?

**Comment by Technyze** [comment, marked as answer]
You would have to do it as a macro. Something like this.
{'battle 2 6', 'macro','input /p I am provoking;input /ja Provoke <t>','',
'Provoke',''},

**Reply by Andorahfal** [reply]
OOO ok, I will try once the server is back up. Thank you
